deploy/olm-catalog/seldon-deploy-operator/0.7.0/seldon-deploy-operator.v0.7.0.clusterserviceversion.yaml

```diff
@@ -175,14 +175,17 @@ metadata:
     capabilities: Seamless Upgrades
     categories: AI/Machine Learning
     certified: "false"
-    containerImage: seldonio/seldon-deploy-operator:0.7.0
+    containerImage: seldonio/seldon-deploy-operator:0.7.0.patch
```

Comment:
This line obv isn't pointing to redhat registry but it is what went out and passed the testing. I think the operator's own image (this one) is always replaced by marketplace. It's the alm-examples ones that we have to make sure we change.